flatten.js

```diff
@@ -0,0 +1,15 @@
+// Problem 3: Complete the flatten function that takes a JS Object, returns a JS Object in flatten format (compressed)
+function flatten(unflatObject) {
+  // Write your code here
+  var outp = {};
+  var x = unflatObject;
```

Comment:
variables are very badly named! Always name the variables according to what they store. eg: `key, unflatObjectClone`, etc